scripts/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+var slideshow = [];
+
+function Slide (slideObject) {
+
+  this.title = slideObject.title;
+  this.caption = slideObject.caption;
+  this.summary = slideObject.summary;
+  this.gitUrl = slideObject.gitUrl;
+};
+
+Article.prototype.toHtml = function() {
```

Comment:
You do not have an object called Article. Is this supposed to relate to your Slide object?